code/obj/item/device/pda2/smallprogs.dm

```diff
@@ -808,6 +808,14 @@ Code:
 
 		src.post_signal(signal)
 
+		if(isliving(usr))
+			playsound(src.master, "sound/items/security_alert.ogg", 60)
+			var/map_text = null
+			map_text = make_chat_maptext(usr, "Emergency alert sent.", "font-family: 'Helvetica'; color: #D30000; font-size: 7px;", alpha = 215)
+			for (var/mob/O in hearers(usr))
```

Comment:
audible message

Reply:
No it's not just a replace, no need to loop

Reply:
I'm definitely not experienced enough with code to know how to do this cleanly, sorry. Give me a little bit

Reply:
it's literally just visible message but for audible